core/src/main/resources/jenkins/model/BuiltInNodeMigration/message.properties

```diff
@@ -1,6 +1,6 @@
 # The MIT License
```

Comment:
FTR the other node_xx.properties were deleted, and one was just renamed. Perhaps a bit weird in isolation but easy to understand with the big picture